include/alpaka/mem/view/ViewPlainPtr.hpp

```diff
@@ -76,7 +76,7 @@ namespace alpaka
                 ViewPlainPtr(ViewPlainPtr const &) = default;
                 //-----------------------------------------------------------------------------
                 ALPAKA_FN_HOST
-                ViewPlainPtr(ViewPlainPtr && other) :
+                ViewPlainPtr(ViewPlainPtr && other) noexcept :
```

Comment:
Are these suggested by the tool? I assume `noexcept` could be added to other places as well, what's your general opinion on that?

Reply:
Yes, this was suggested. I do not have anything against more `noexcept`.